webui/react/src/pages/JobQueue/JobQueue.settings.ts

```diff
@@ -22,6 +23,11 @@ const config: SettingsConfig = {
       storageKey: 'sortKey',
       type: { baseType: BaseType.String },
     },
+    {
+      key: 'selectedRp',
+      storageKey: 'selectedRp',
```

Comment:
**nit:** since this is going to be exposed to the local storage and possibly in the db in the future, what do you think about something more descriptive of `selectedPool` instead? I think any references of `pool` can currently be assumed that it is a resource pool we are talking about.